pages/configuration/elasticsearch.rst

```diff
@@ -66,7 +66,6 @@ The following configuration options are now being used to configure connectivity
 Automatic node discovery
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. caution:: Automatic node discovery will be available starting with Graylog 2.3.0-alpha.3/beta.1.
```

Comment:
We should also add a warning that the automatic node discovery *does not* support authentication.